tests/SampleApps/python/python2-rest-framework-app/requirements.txt

```diff
@@ -92,7 +92,7 @@ chardet==3.0.4
 Click==7.0
 coreapi==2.3.3
 coreschema==0.0.4
-cryptography==2.4.2
+cryptography==39.0.1
```

Comment:
I think this change is coming from a `dependabot` PR -- you may want to rebase your PR against `main` so that the changes that appear are only the ones that you've made and that are different from what's in the `main` branch.